core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -672,6 +679,13 @@ public String toStringRequestOnly() {
 
     @Override
     public String toStringResponseOnly() {
+        return toStringResponseOnly(Function.identity(), Function.identity());
+    }
+
+    @Override
+    public String toStringResponseOnly(Function<HttpHeaders, HttpHeaders> headersSanitizer,
+                                       Function<Object, Object> contentSanitizer) {
```

Comment:
Should the sanitizers be `Function<? super HttpHeaders, ?>` and `Function<Object, ?>` instead?